pkg/provider/akeyless/akeyless.go

```diff
@@ -71,6 +71,33 @@ func (p *Provider) NewClient(ctx context.Context, store esv1beta1.GenericStore,
 }
 
 func (p *Provider) ValidateStore(store esv1beta1.GenericStore) error {
+	storeSpec := store.GetSpec()
+	akeylessSpec := storeSpec.Provider.Akeyless
+
+	akeylessGWApiURL := akeylessSpec.AkeylessGWApiURL
+
+	if akeylessGWApiURL == nil {
+		return fmt.Errorf("Akeyless GW API URL is required ")
+	}
+
+	accessID := akeylessSpec.Auth.SecretRef.AccessID
+	err := utils.ValidateSecretSelector(store, accessID)
+	if err != nil {
+		return err
+	}
```

Comment:
You also need to check for accessID.Name and accessID.Key not being empty, as they are being used by this method https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/akeyless/auth.go#L36

Reply:
Alright, got it. I will add this.